dune_client/client.py

```diff
@@ -238,3 +252,125 @@ def refresh_into_dataframe(
             ) from exc
         data = self.refresh_csv(query, performance=performance).data
         return pandas.read_csv(data)
+
+    # CRUD Operations: https://dune.com/docs/api/api-reference/edit-queries/
+    def create_query(
+        self,
+        name: str,
+        query_sql: str,
+        params: Optional[list[QueryParameter]] = None,
+        is_private: bool = False,
+    ) -> int:
```

Comment:
I could maybe do a type alias for this integer.

Reply:
Or I could just add to the doc string that the function returns the query id...

Reply:
That sounds like a great idea!

Reply:
I reworked this and think it turned out pretty well. We now have this `DuneQuery` structure with `QueryBase` as the old `Query` and `QueryMeta` containing a bunch of meta content. the `raw_sql`is a stand alone field on the `DuneQuery` class. A couple of things to note here are

- that `is_private` might actually belong more naturally on `QueryBase` since then this class would contain all the fields used for query creation.
- renaming `Query` to `QueryBase` is a breaking change that would require a full version bump (i.e. anyone using the package would have to update their code when importing this Query class. Not sure if we wanted to go this route or not.

How do you feel  about all of this.

Reply:
Sounds good to me.